### DISCUSSION TITLE
Loss function

### DISCUSSION THREAD

**Discussion by SutirthaChakraborty** [hmartelb/avlit#8] [General]
How did you do the loss function ?

**Comment by hmartelb** [comment]
Please refer to the section "3.3 Implementation details" in our paper. There we state that "The training objective is the negative SI-SDR [30]". You can find such metric in many open-source repositories, although this one only includes the model architecture. Regards.

**Comment by SutirthaChakraborty** [comment]
Just curious, won't you share the training code?

**Reply by hmartelb** [reply]
Moved this thread to discussion, according to [Readme](https://github.com/hmartelb/avlit#contact) since this is not about a bug. 
Answering your question, there are no plans to share the training code because it adds unnecesary complexity to the repository. The main contribution is the architecture and we want to focus on that. There is nothing special/different in the way the model is trained, and all the parameters are listed in the paper.
